guides/v2.1/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -96,6 +96,20 @@ CSS source maps solve this issue. They help to find the `.less` file, where the
 
 CSS source maps are generated automatically when you compile CSS for your theme using the `grunt less: <theme>` command. To use them, you need to enable source maps displaying in your browser.
 
+Magento 2 has a base set of Variables that define commonly used aspects of a theme. Colors, fonts, style of page titles, and so on are defined in these variables. 
+
+In `<magento-root>/lib/web/css/source/lib/variables` you will find a number of intuitively named LESS files. In each of these you can find values assigned to variables for many of the common elements in Magento 2. 
+
+To change/override any of these variables, simply create a file in `<theme-dir>/web/css/source/_theme.less` For example:
+
+```
+@navigation__background: @secondary__color__light; 
+@font-family__sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif;
+```
+![node declaration autocomplete]({{ site.baseurl }}/common/images/fdg/lib-map.png){:width="610px"}
+
+CSS source maps solve this issue. They help to find the .less file from the lib directory, where the style is specified. 
+
 The path to the CSS source maps configuration differs, depending on the browser.
 
 In Google Chrome, to enable source maps generation, go to **Inspect** > **Settings** > **Preferences** > **Enable CSS source maps**.   
```

Comment:
I think your new section should appear after this section.

Reply:
I have moved my section as per suggestion.